### DISCUSSION TITLE
一加社区 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#55] [模板讨论]
一加社区 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by bgaidu** [comment]
----连续签到：【】天，月累计签到：【】天，累计签到：【】天，签到等级：【】，加油奖励：【】
日志显示如下 盼修复

**Reply by wjf0214** [reply]
一加的签到已经下线了。
我之前说观望一阵，没删。
已经没用了。我待会就删除
